src/site/markdown/server/manage-webadmin.md

```diff
@@ -1386,6 +1387,94 @@ Response codes:
  - 204: OK
  - 400: Alias structure or member is not valid
 
+## Creating address domain
+
+You can use **webadmin** to define domain mappings.
+
+Given a configured source (from) domain and a destination (to) domain, when an email is sent to an address belonging to the source domain, then the domain part of this address is overwritten, the destination domain is then used.
+A source (from) domain can have many destination (to) domains. 
+
+For example: with a source domain `james.apache.org` maps to two destination domains `james.org` and `apache-james.org`, when a mail is sent to `admin@james.apache.org`, then it will be navigated two addresses `admin@james.org` and `admin@apache-james.org`
+
+This feature uses [Recipients rewrite table](/server/config-recipientrewritetable.html) and requires
+the [RecipientRewriteTable mailet](https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java)
```

Comment:
we could link to the documentation instead (https://james.apache.org/server/config-recipientrewritetable.html)

Reply:
the link you provided is already given in the statement above.

Reply:
oh, sorry